istioctl/pkg/multixds/gather.go

```diff
@@ -109,3 +112,62 @@ func makeSan(istioNamespace, revision string) string {
 	}
 	return fmt.Sprintf("istiod-%s.%s.svc", revision, istioNamespace)
 }
+
+// AllRequestAndProcessXds returns all XDS responses from 1 central or 1..N local XDS servers
```

Comment:
What is a local XDS Server?

Reply:
By "local" I meant one in a cluster under user control where we can list pods and port-forward into them.  Poor choice of words.

Reply:
Fixed.